known-limitations.md

```diff
@@ -0,0 +1,181 @@
+~~~
+
+~~~
++------------------------+
+| 'string1' || 'string2' |
++------------------------+
+| string1string2         |
++------------------------+
+(1 row)
+~~~
+
+~~~ sql
+> SELECT ('string1' collate en) || ('string2' collate en);
+~~~
+
+~~~
+pq: unsupported binary operator: <collatedstring{en}> || <collatedstring{en}>
+~~~
+
+## Creating views with array types
+
+It's not possible to [create a view](create-view.html) with an array type in the view's `SELECT` query. Attempting to do so causes the node that receives the request to crash.
+
+## Write and update limits for a single transaction
+
+A single transaction can contain at most 100,000 write operations (e.g., changes to individual columns) and at most 64MiB of combined updates. When a transaction exceeds these limits, it gets aborted.
+
+If you need to increase these limits, you can update the [cluster-wide settings](cluster-settings.html) `kv.transaction.max_intents` and `kv.raft.command.max_size`. However, we strongly encourage you to discuss your goals with CockroachDB before doing so.
+
+## Max size of a single column family
+
+When creating or updating a row, if the combined size of all the values in a single [column family](column-families.html) exceeds the max range size (64MiB by default) for the table, the operation may fail, or cluster performance may suffer.
+
+As a workaround, you can either [manually split a table's columns into multiple column families](column-families.html#manual-override), or you can [create a table-specific zone configuration](configure-replication-zones.html#create-a-replication-zone-for-a-table) with an increased max range size.
+
+## Simultaneous client connections and running queries on a single node
+
+When a node has both a high number of client connections and running queries, the node may crash due to memory exhaustion. This is due to CockroachDB not limiting the number of clients and queries based on how much free RAM is available on the node.
+
+To prevent memory exhaustion, monitor each node's memory usage and ensure there is some margin between maximum CockroachDB memory usage and available system RAM. For more details about memory usage in CockroachDB, see [this blog post](https://www.cockroachlabs.com/blog/memory-usage-cockroachdb/).
+
+## SQL subexpressions and memory usage
+
+Many SQL subexpressions (e.g., `ORDER BY`, `UNION`/`INTERSECT`/`EXCEPT`, `GROUP BY`, subqueries, window functions) accumulate intermediate results in RAM on the node processing the query. If the operator attempts to process more rows than can fit into RAM, either a memory capacity error will be reported to the client that issued the query, or the node may crash. For more details about memory usage in CockroachDB, see [this blog post](https://www.cockroachlabs.com/blog/memory-usage-cockroachdb/).
+
+## Counting distinct rows in a table
+
+When using `count(DISTINCT a.*)` to count distinct rows in a table, the results are almost always incorrect, for example:
```

Comment:
" ... to count distinct rows in a table based on a subset of the columns, as opposed to `count(*)`, the results ..."

the rest lgtm